packages/core/config/jest.config.web.js

```diff
@@ -2,11 +2,29 @@
 
 const path = require('path')
 
+const { getPaths } = require('@redwoodjs/internal')
+
+const redwoodPaths = getPaths()
+
 module.exports = {
   resolver: 'jest-directory-named-resolver',
   rootDir: process.cwd(),
   globals: {
     __REDWOOD__API_PROXY_PATH: '/',
   },
   setupFilesAfterEnv: [path.resolve(__dirname, './jest.setup.web.js')],
```

Comment:
I know you didn't change this, but I wanted to make a note for myself to change this.
```suggestion 
  // TODO: use getPaths().web.base instead.
  setupFilesAfterEnv: [path.resolve(__dirname, './jest.setup.web.js')],
```